iOS/src/toga_iOS/widgets/slider.py

```diff
@@ -81,7 +81,7 @@ def get_value(self):
 
     def set_value(self, value):
         self.value = value
-        self.native.setValue(value, animated=True)
+        self.native.setValue_animated_(value, True)
```

Comment:
I'm definitely still a noob about Rubicon and ObjC... what's the functional difference between these?